documentation/typespec-rest-api-dev-process.md

```diff
@@ -78,27 +78,26 @@ To start, you just need to browse to the `azure-rest-api-specs` repo, select `<>
 
 ### 3. Creating a new TypeSpec project
 
-Please first review recommended folder structure detailed in [this document](https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/typespec-structure-guidelines.md).
+Please first review recommended folder structure detailed in [this document](https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/directory-structure.md).
 
-1. Under `[reporoot]\specifications`, create service folder directly.
-2. Create your service component folder under the service folder. For example, `Sphere.Management` or `Azure.OpenAI`.
-3. Create a new TypeSpec project based on Azure template with command:
+1. Under `[reporoot]\specification`, create service folder directly.
+   - Select `(rest-api-spec repo) ARM` or `(rest-api-spec repo) Data-plane` and answer appropriate naming questions.
+   - If it's ARM service, the service folder would be `specification/{organization}/resource-manager/<RPNS>/{service}`.
+   - If it's Data Plane service, the service folder would be `specification/{organization}/data-plane/{service}`
+2. Create a new TypeSpec project based on Azure template with command:
 
     ```cli
        npx tsp init https://aka.ms/typespec/azure-init
```

Comment:
@markcowl we need to fix this issue https://github.com/Azure/typespec-azure/issues/3425 before this documentation update PR gets merged.